compair/static/modules/assignment/assignment-module.js

```diff
@@ -959,9 +959,7 @@ module.controller("AssignmentViewController",
                         $scope.assignment.status.answers.answered = $scope.assignment.status.answers.count > 0;
                     }
                     $scope.updateAnswerList();
-                    $location.hash(null);
```

Comment:
Fixes #705

Reply:
Can you put all the misc fixes together into a separate PR (to help keep track of things), thanks